src/easyscore.ts

```diff
@@ -350,7 +354,15 @@ export class Builder {
 
     // Build StaveNotes.
     const { chord, duration, dots, type } = this.piece;
-    const keys: string[] = chord.map((notePiece) => notePiece.key + '/' + notePiece.octave);
+    const keys: string[] = chord.map(
+      (notePiece) =>
+        notePiece.key +
+        (notePiece.accid && ['bb', 'b', '##', '#', 'n'].includes(notePiece.accid.toLowerCase())
```

Comment:
Can you make this:

```typescript
const keys: string[] = chord.map((notePiece) => notePiece.key + ['bb', 'b', '##', '#', 'n'].includes(notePiece.accid) ? notePiece.accid : '' + '/' + notePiece.octave);
```

No need for `toLowerCase` because the parser already restricts case, and no need for the extra `notePiece.accid &&`.

Reply:
`notePiece.accid` can be `undefined` and `notePiece.accid &&` was required in order to avoid the complain in `.includes(notePiece.accid)`

I changed to `.includes(notePiece.accid ?? '')` which is more compact.